ui-tests/src/test/java/org/openmrs/reference/ContactInfoTest.java

```diff
@@ -36,7 +36,6 @@ public void setUp() throws Exception {
     @Test
     @Category(BuildTests.class)
     public void contactInfoTest() throws Exception {
-    
```

Comment:
What is the use of this empty line?

Reply:
This line has no use at all, i used it in response on empty commit but i was to revert it back